rpy2/robjects/help.py

```diff
@@ -262,10 +262,10 @@ def usage(self) -> str:
         """ Get the usage for the object """
         return self._get_section(r'\usage')
 
-    def iteritems(self):
+    def items(self):
         """ iterator through the sections names and content
         in the documentation Page. """
-        return self.sections.iteritems
+        return self.sections.items
```

Comment:
`self.sections` must be of type `collections.OrderedDict`, IIRC. This class no longer has a method `iteritems()`. There should have been a unit test to catch that.